draft-ietf-rats-architecture.md

```diff
@@ -131,7 +131,7 @@ Attestation Result:
 
 Attester:
 
-: An entity (typically a device), whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation
+: An entity whose attributes must be appraised in order to determine whether the entity is considered trustworthy, such as when deciding whether the entity is authorized to perform some operation
```

Comment:
I believe "Evidence" is slightly better than "attributes" (but I can live with it either way) since only the attributes that appear in Evidence are actually appraised.

On "whether" vs "the extent to which", I believe the latter is more correct since the former implies it's always exactly boolean, and the latter is more liberal in terms of how may different possibilities there are.

Reply:
+1, I agree with Dave

Reply:
This change has been reverted.